views/dashboard.handlebars

```diff
@@ -1,4 +1,26 @@
 {{> nav}}
 
+<div class="container">
+    <div class="hero ">
+        <div class="hero-body" >
+            <img src="/images/gamers_game.png" alt="banner with gaming consol controllers">
+        </div>
+    </div>
+    <div class="hero is-primary has-text-black is-medium">
+        <div class="hero-body has-text-centered">
+            <h1 class="title"> Hello {{fetchName}} !</h1>
```

Comment:
fetch name is a misleading variable. it imples that youre going to fetch something. it should be name of full name

Reply:
Renamed 
userDBname, userDBusername